docs/src/dev/report_gen.md

```diff
@@ -26,17 +26,17 @@ include("report_gen.jl")
 
 ## Errors with a Limiter
 
-In order to verify the correctness of our algorithms with a limiter, we recreate Table 1 from ["Optimization-based limiters for the spectral element method" by Guba et al.](https://www.sciencedirect.com/science/article/pii/S0021999114001491) This involves running the `horizontal_deformational_flow` test case (from ["A standard test case suite for two-dimensional linear transport on the sphere" by Lauritzen et al.](https://gmd.copernicus.org/articles/5/887/2012/gmd-5-887-2012.pdf)) with and without a limiter, and also with and without hyperdiffusion. This test case uses a limited tendency `T_lim!` (which consists of advection and, optionally, hyperdiffusion), along with `dss!` and `lim!`. The spatial discretization is implemented using `ClimaCore`. Since this analysis is relatively expensive to run, we only check the results for `SSP333` and `ARS343`. Note that it is possible to limit undershoots and overshoots to 0 (up to floating-point roundoff error) when using the `SSPConstrained` `SSP333`, but not when using the `Unconstrained` `ARS343`.
+In order to verify the correctness of our algorithms with a limiter, we recreate Table 1 from ["Optimization-based limiters for the spectral element method" by Guba et al.](https://www.sciencedirect.com/science/article/pii/S0021999114001491) This involves running the `horizontal_deformational_flow` test case (from ["A standard test case suite for two-dimensional linear transport on the sphere" by Lauritzen et al.](https://gmd.copernicus.org/articles/5/887/2012/gmd-5-887-2012.pdf)) with and without a limiter, and also with and without hyperdiffusion. This test case uses a limited tendency `T_lim!` (which consists of advection and, optionally, hyperdiffusion), along with `dss!` and `lim!`. The spatial discretization is implemented using `ClimaCore`. Since this analysis is relatively expensive to run, we only check the results for `SSP333` and `ARS343`. Note that it is possible to limit undershoots and overshoots to 0 (up to floating-point roundoff error) when using the `SSP` `SSP333`, but not when using the `Unconstrained` `ARS343`.
 
 ```@example
-using ClimaTimeSteppers # hide
-ENV["GKSwstype"] = "nul" # hide
-include(joinpath(@__DIR__, "..", "plotting_utils.jl")) # hide
-include(joinpath(pkgdir(ClimaTimeSteppers), "test", "problems.jl")) # hide
-
-# This also runs with num_steps = 1000, but with larger under/overshoots; 4800
-# is the value used in the paper.
-limiter_summary(Float64, [SSP333(), ARS343()], horizontal_deformational_flow_test, 4800)
+# using ClimaTimeSteppers # hide
+# ENV["GKSwstype"] = "nul" # hide
+# include(joinpath(@__DIR__, "..", "plotting_utils.jl")) # hide
+# include(joinpath(pkgdir(ClimaTimeSteppers), "test", "problems.jl")) # hide
+# 
+# # This also runs with num_steps = 1000, but with larger under/overshoots; 4800
+# # is the value used in the paper.
+# limiter_summary(Float64, [SSP333(), ARS343()], # horizontal_deformational_flow_test, 4800)
```

Comment:
Was this meant to be commented?

Reply:
Ah, shoot, it was not. Thanks for catching this!